pkg/apis/application/v1alpha1/types.go

```diff
@@ -181,16 +181,18 @@ const (
 
 // ApplicationSourceHelm holds helm specific options
 type ApplicationSourceHelm struct {
+	// Version is the Helm version to use for templating with
+	Version string `json:"version,omitempty" protobuf:"bytes,1,opt,name=version"`
```

Comment:
For backwards compatibility, please make the new member the last field of protobuf (i.e. prepend, not append)

Reply:
Roger! Will fix!

Reply:
Fixed in https://github.com/argoproj/argo-cd/commit/cbb5b270f72edc75ba8e61b8fe4fc6f0dae0ca3f